apps/src/applab/commands.js

```diff
@@ -1578,16 +1581,40 @@ function filterUrl(urlToCheck) {
   })
     .success(data => {
       let approved = data['approved'];
-      console.log('Url determined safe to open: ' + approved);
+      getStore().dispatch(actions.addRedirectNotice(approved, urlToCheck));
     })
     .fail((jqXhr, status) => {
-      console.log('Error. Please re-run program');
+      // When this query fails, default to the dialog that allows the user to choose
+      getStore().dispatch(actions.addRedirectNotice(true, urlToCheck));
```

Comment:
Perhaps this isn't the right place to raise this concern, but has there been any discussion on whether we should allow them to continue on if the ajax request fails? My instinct is to block unless we can verify the safety of the site rather than allow unless we can prove the unsafety of the site.

Reply:
Totally the right place to raise this concern, in my opinion. :) Ryan has an explanation in the [spec](https://docs.google.com/document/d/1cW1OiVLmXhAzBU4MSNM0q6g3UA5zZTZapQgUMxp3V84), but a quick summary is that our 'approval' dialog still says (paraphrased), "You are going here. Are you sure?", so it allows the user to make an informed choice to move forward in cases where we do not have any additional information to provide.